packages/dialtone-css/lib/build/less/components/skeleton.less

```diff
@@ -17,15 +17,19 @@
 //  ============================================================================
 //  @   SKELETON CONTAINER
 //  ----------------------------------------------------------------------------
+
+// The --placeholder-from-color and --placeholder-to-color
+// custom properties can be set on the parent class of the
+// placeholder to control the animation colors.
 .skeleton-placeholder {
   --placeholder-from-color: hsla(var(--dt-color-black-900-hsl) / 0.2);
   --placeholder-to-color: hsla(var(--dt-color-black-900-hsl) / 0.1);
 
   display: flex;
   width: 100%;
-  background: var(--placeholder-from-color);
+  background: var(--placeholder-from-color, var(--dt-color-black-300));
```

Comment:
Let's make the fallback color `var(--dt-color-surface-bold-opaque)`. Same for the others below.

Reply:
What about the `--placeholder-to-color`? currently falls back to `var(--dt-color-black-100)`

Reply:
Ah yes, use `var(--dt-color-surface-moderate-opaque)`. That doesn't map to the current Black 100, but it's a less jarring animating shift. 

Also, remove the...
```
style="
  ...
  --placeholder-from-color: ...;
  --placeholder-to-color: ...;
  ...
"
```
...from the `## Prefabricated examples`.

Reply:
Maybe belay that, look at [this other comment](https://github.com/dialpad/dialtone/pull/557#pullrequestreview-2409386555) first.

But still do the removal of `style="..."` comment above.

Reply:
Alright, should be good.